xlsynth-g8r/fuzz/fuzz_targets/fuzz_ir_outline_equiv.rs

```diff
@@ -189,7 +189,7 @@ fuzz_target!(|sample: FuzzSample| {
     }
     if !has_boundary_output(&orig_fn, &region) {
         return; // Early-return: selected region produces no boundary outputs
-                // (uninformative)
+        // (uninformative)
```

Comment:
Maybe hoist the comment here above the return.

Reply:
done.